Sources/_StringProcessing/Regex/AnyRegexOutput.swift

```diff
@@ -229,17 +226,14 @@ extension AnyRegexOutput {
     /// `Substring` has optional depth `0`, and `Int??` has optional depth `2`.
     let optionalDepth: Int
 
-    /// The bounds of the output element.
-    let bounds: Range<String.Index>?
+    /// The element bounds and the value (if available).
+    let rawValue: (range: Range<String.Index>, value: Any?)?
```

Comment:
The name `rawValue` is used pretty heavily with `RawRepresentable`, but IIUC this tuple alone isn't sufficient for constructing the type. Any particular reason we're storing the tuple instead of having a `range` and `value` field?

Reply:
The name is definitely up for debate; any suggestions?

I made it a tuple because when `range`  is non-nil, `value` must be non-nil.  This is so that we can avoid asserting `value != nil` in a bunch of places.  I could also make `range` and `value` be a struct.

Reply:
Renamed the property to `content`, as it represents the content of the capture, whereas other properties like `optionalDepth`, `name`, `referenceID` are metadata.